velox/dwio/common/tests/utils/FilterGenerator.cpp

```diff
@@ -165,6 +165,40 @@ std::unique_ptr<Filter> ColumnStats<StringView>::makeRangeFilter(
     return std::make_unique<velox::common::IsNull>();
   }
 
+  int32_t lowerIndex;
```

Comment:
THe commit title is too long. It should be within 50 chars. Consider just use "rename ColumnStats<StringView>::makeRangeFilter"

Also, include in the commit message why this changes was made.